pop-api/integration-tests/src/fungibles/mod.rs

```diff
@@ -364,8 +382,14 @@ fn instantiate_and_create_fungible_works() {
 			Err(Module { index: 52, error: 5 })
 		);
 		// Successfully create an asset when instantiating the contract.
-		assert_ok!(instantiate_and_create_fungible(contract, ASSET_ID, 1));
+		let instantiator =
+			instantiate_and_create_fungible(contract, ASSET_ID, 1).expect("Should work");
```

Comment:
Could we remove the `expect` and make sure we `assert_ok` it and separately get contract address pls